slam-ws/src/caffeine/launch/caffeine_gazebo.launch

```diff
@@ -29,4 +29,10 @@
 	<!-- Set up robot_localization Odometry -->
 	<include file="$(find odom)/launch/odom.launch"/>
 
+	<!-- Add Lidar Filtering -->
```

Comment:
Can you put this above the `odom.launch`, for consistency on how the `caffeine_setup.launch` will be organized